index.d.ts

```diff
@@ -1156,7 +1156,11 @@ declare namespace Eris {
   export class Collection<T extends { id: string | number }> extends Map<string | number, T> {
     baseObject: new (...args: any[]) => T;
     limit?: number;
-    constructor(baseObject: new (...args: any[]) => T, limit?: number);
+    constructor(baseObject: new (...args: any[]) => T, limit?: number, iterable?: object | any[]);
+    static from(array: any[], key: string, limit: number): Collection<any>;
```

Comment:
```suggestion
    static from(array: T[], key: string, limit: number): Collection<T>;
```

Reply:
This doesn't work because it is static (I guess?).
TS on vscode doesn't seem to be happy with it at least?